web-src/src/components/AdditionalContextInput.js

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2023 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+import {
+  ActionButton, ProgressCircle, TextArea, TextField,
+} from '@adobe/react-spectrum';
+import React, { useCallback, useState } from 'react';
+import { css } from '@emotion/css';
+import { ToastQueue } from '@react-spectrum/toast';
+import { useApplicationContext } from './ApplicationProvider.js';
+import { DescriptionLabel } from './DescriptionLabel.js';
+
+const DEFAULT_PROMPT = 'Summarize the key points from the following source document to guide the creation of the content';
+const DEFAULT_SELECTOR = 'p, h1, h2, h3, h4, h5, h6, li, td, th, span, blockquote, article, section, header, footer, aside, nav, div:only-child:empty';
```

Comment:
@vtsaplin , Just curious, what is the purpose of having `div:only-child:empty`?

Reply:
Good catch! It's leftover from the negative pattern. Let's remove it.